fontbe/src/orchestration.rs

```diff
@@ -219,8 +221,91 @@ impl From<&VariationRegion> for TupleBuilder {
     }
 }
 
+#[repr(u8)]
+#[derive(Copy, Clone, Debug, PartialEq)]
+pub enum LocaFormat {
+    Short = 0,
+    Long = 1,
+}
+
+impl LocaFormat {
+    pub fn new(loca: &[u32]) -> LocaFormat {
+        // https://github.com/fonttools/fonttools/blob/1c283756a5e39d69459eea80ed12792adc4922dd/Lib/fontTools/ttLib/tables/_l_o_c_a.py#L37
+        if loca.last().copied().unwrap_or_default() < 0x20000
+            && loca.iter().all(|offset| offset % 2 == 0)
```

Comment:
I wonder if it's really worth serializing this single-byte loca.format to disk (along with all the accompanying logic that this PR adds), when it can be computed using this simple formula based only on the number of locations and whether they are all 2-byte padded?

Reply:
oh right, if we want to persist the loca table itself, we need to be able to deserialize it and for that we need the format. nevermind